activerecord/test/cases/insert_all_test.rb

```diff
@@ -335,6 +335,11 @@ def test_upsert_all_works_with_partitioned_indexes
                  Measurement.where(city_id: 2).pluck(:logdate, :peaktemp, :unitsales)
   end
 
+  def test_insert_all_with_enum_values
+    Book.insert_all [{ status: :published, isbn: "1234566" }, { status: :proposed, isbn: "1234567" }]
+    assert Book.where(isbn: ["1234566", "1234567"]).pluck(:status), [:published, :proposed]
```

Comment:
```suggestion
    assert_equal ["published", "proposed"], Book.where(isbn: ["1234566", "1234567"]).pluck(:status)
```

`assert` passes if its first argument is truthy, which will always be the case here - this test as written can't fail.

Reply:
@eugeneius you're 100% right